### ISSUE TITLE
TypeError: Cannot read properties of null

### ISSUE THREAD

**Issue by uasan** [porsager/postgres#377]
postgres version: 3.2.3

```JS
await sql`SELECT ${['A', null, 'B']}::text[]`;
```

Error log
```
TypeError: Cannot read properties of null (reading 'type')
    at node_modules/postgres/src/types.js:247:49
```

**Comment by porsager** [comment]
Hi @uasan - long time ago - how are you doing? :)

This is fixed in master, and I'll make a release later today.

**Comment by uasan** [comment]
> Hi @uasan - long time ago - how are you doing? :)

Normal, except that in my home country of Ukraine, there is a war now.

> This is fixed in master, and I'll make a release later today.

Ok, thanks.

**Comment by porsager** [comment]
Sorry man! Hope you and your family are safe! Best wishes!

I've released v3.2.4 with the fix now.
